### DISCUSSION TITLE
Settare data input_datetime per mezzo di script

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#1814] [HUB: Home Assistant]
Sto creando uno script che viene eseguito alla pressione di un pulsante.
Lo script esegue diverse azioni una delle quali dovrebbe cambiare il valore di una input_datetime settata per mostrare sola la data. Nelle mie intenzioni la data da mostrare è il 15 settembre dell'anno corrente. Ad esempio se eseguito oggi sarebbe "2022-09-15", se premuto il prossimo anno "2023-09-15".
Sapete darmi qualche dritta su come compilare il campo Date dello script? Ho fatto diverse prove ma eseguendo lo script questo finisce sempre per avere un errore si parser relativa alla data.

**Comment by 50494554524F** [comment]
https://www.home-assistant.io/integrations/input_datetime/#services
